src/components/BookList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import BookItem from './BookItem';
+import BookAdd from './BookAdd';
+
+function BookList() {
+  return (
+    <>
+      <ul>
+        <BookItem />
+      </ul>
+      <BookAdd />
+    </>
+  );
+}
```

Comment:
- [ ] It will be great to have some dummy data of your book so that you can loop the book item and pass in the book data as props. This will make your application, even more, data ready as specified in the project specifications.
 Example: ```const list = [ 
      { "name": "firstBook", "author": "keylemba", id: 1},
   { "name": "secondbook", "author": "keylemba", id: 2}
 ]```

Then using the data above loop through the `bookItem` component and pass the dummy data as props

Reply:
done. Thank you @peterrobert